cpp/src/core/sroptcryst.h

```diff
@@ -828,8 +810,7 @@ class srTOptCryst : public srTGenOptElem {
 		return 0;
 	}
 
-	void RadPointModifier(srTEXZ& EXZ, srTEFieldPtrs& EPtrs, void* pBufVars=0) //OC29082019
-	//void RadPointModifier(srTEXZ& EXZ, srTEFieldPtrs& EPtrs)
+	void RadPointModifier(srTEXZ& EXZ, srTEFieldPtrs& EPtrs)
```

Comment:
Left the previous version (at least for the moment). Even though "void* pBufVars" parameter is not used here, it is defined in the corresponding virtual function in the base class (in attempt to fix multithreading issues in some places). The list of arguments may need to be the same here as in the original definition of the virtual function.